codes/quantum/qudits/holographic.yml

```diff
@@ -0,0 +1,36 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: holographic
+physical: qudits
+logical: qudits
+
+
+name: 'Holographic code'
+introduced: '\cite{arxiv:1503.06237}'
+
+description: 'Encodes operators in the bulk of the Anti de Sitter space (analogous to logical qudits) into operators on the boundary of the corresponding Conformal Field Theory (analagous to physical qudits). In general, this encoding is an isometry \(\mathcal{V}\) that maps the hilbert space of the AdS to the hilbert space of the boundary CFT. Furthermore, \(\mathcal{V}\) obeys the entanglement wedge reconstruction condition, which states that, for any boundary region \(R\), any bulk operator localized to the entanglement wedge of R (let us denote this as \(O\)) must be implementable by a boundary operator localized to R (let us denote this as \(\tilde{O}\)). Formally, \(\mathcal{V}O = \tilde{O}\mathcal{V}\) and \([\tilde{O},\mathcal{V}\mathcal{V}^\dagger] = 0\). The entanglement wedge is the space enclosed within the Ryu–Takayanagi surface in the bulk (minimal surface) with boundary R.'
+
+protection: 'Protects against erasure errors on the boundary.'
+
+
+features:
+  decoders:
+    - 'Given only a fraction of the boundary operators, one can only decode (reconstruct) a portion of the bulk operators within the entanglement wedge.'
+   
+  transversal_gates:
+    - 'There exist holographic codes with arbitrary transversal gate sets for any compact Lie group \cite{arxiv:2108.11402}' 
+    
+  threshold:
+    - 'The ideal holographic code (perfect representation of AdS/CFT) should be able to protect a central bulk operator against \(50%\) erasure rate on the boundary in line with AdS-Rindler reconstruction. \cite{arxiv:1503.06237}'
+
+
+notes:
+    - 'All Boundary global symmetries must be dual to Bulk gauge symmetries, and vice versa \cite{arxiv:1810.05338v2}.'
+
+relations:
+  parents:
```

Comment:
Why oaecc? Are they protecting algebras? If there are gauged versions of these codes, then oecc (which is a child of oaecc) is sufficient I believe. If there are no subsystem/gauge versions, then qecc is fine.